src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -330,7 +330,7 @@ def __get_test_list_by_runtime(self, test_unit = constants.TEST_UNIT_NAME):
                     if constants.DISABLED_PREFIX in real_test_name:
                         self._disabled_tests.append(real_test_name)
                     elif test_unit == constants.TEST_UNIT_NAME:
-                        tests_dict[real_test_name] = 1
+                        tests_dict[real_test_name] = -1
```

Comment:
what is the purpose of this change? Those change needed to speedup cache creation.

Reply:
@iefode, could you please elaborate on that change? I'm afraid that I don't remember the exact reasoning.

Reply:
@mryzhov You have set the possible execution time. It means that tests is started to be united to processes (but it is mistake). It is not expected by default case. The default approach for `Test` split unit (without cache) is one test per one process. -1 provides the correct distribution.

Reply:
Why it is mistake? If the test have never been executed (it is not in the cache) why we should run in in separate process?

Reply:
@mryzhov I have shared the details how script should work offline and the above comment. I don't understand your point.

Reply:
never mind if it is already merged